app/views/application/_completed_tasks_table.html.erb

```diff
@@ -0,0 +1,10 @@
+<h2>Completed Tasks</h2>
+<div class='tasklist'>
+  <table>
+     <% @completed_tasks.each do |task| %>
+       <tr>
+         <td id='task'><%= link_to task.name, "/tasks/#{task.id}" %></td><td id='delete'><%= render partial: 'delete_button', locals: {id: task.id, controller: 'tasks'} %></td>
```

Comment:
By using an ID here inside of an `each` loop you will end up setting the same ID to multiple `td` elements which is not good. Consider using a class here instead.